docs/content/docs/contributor-docs/release.md

```diff
@@ -61,40 +61,56 @@ Kubernetes Deployment; the Deployment's Pod image will refer to the exact
 Docker image tag matching the release tag.
 
 ## Release steps
-<// mkdocs does not support numbered lists with code blocks. So use 1) instead of 1. in numbered list. >
-1) First check out a git branch for your release:
+
+0) Rebase $SERVICE-controller repo with latest code
+```bash
+# ACK_SOURCE_ROOT is typically $GOSRC/github.com/aws-controllers-k8s
+cd $ACK_SOURCE_ROOT
+cd $SERVICE-controller
+git fetch --all --tags
+# Optionally fetch and rebase the latest code generator
+cd ../code-generator
+git checkout main && git fetch --all --tags && git rebase upstream/main
+```
+
+1) Navigate to $SERVICE-controller repo and check out a git branch for your release:
 ```bash
+cd ../$SERVICE-controller
 export RELEASE_VERSION=v0.0.1
 git checkout -b release-$RELEASE_VERSION
+git branch --set-upstream-to=origin/main release-$RELEASE_VERSION
 ```
 
-2) Build the release artifacts for the controllers you wish to include in the
-   release
+2) Navigate to code-generator repo and build the release artifacts for the controllers you wish to include in the
+   release. *Note, if executing in a terminal separate from $SERVICE-controller repo, `$RELEASE_VERSION` will need to be set again*
 
    Run `make build-controller` for each service from code-generator repository.
     For instance, to build release artifacts for the SNS and S3 controllers I
     would do:
 
 ```bash
+cd ../code-generator
 for SERVICE in sns s3; do
     export SERVICE;
     echo "building ACK controller for $SERVICE, Version: $RELEASE_VERSION"
     make build-controller;
```

Comment:
I actually run `./scripts/build-controller-release $SERVICE` instead, which doesn't rebuild the controller and only regenerates the release artifacts.

Reply:
Truee, this definitely works. My thinking here is if the user decides to rebase code-generator in earlier steps, then they will need to regen the controller so I'm including the command that covers both (controller and release artifacts generation) to be safe

Reply:
While we are commenting on this, could we change this from a for loop, looping over `sns` and `s3`, into just a single call to `make build-controller` for a given `$SERVICE`. I don't think most developers are going to be releasing multiple controllers at once, anyway.